src/entity/posts/FreeformPost.ts

```diff
@@ -0,0 +1,14 @@
+import { ChildEntity, Column } from 'typeorm';
+import { Post, PostType } from './Post';
+
+@ChildEntity(PostType.Freeform)
+export class FreeformPost extends Post {
+  @Column({ type: 'text', nullable: true })
+  image?: string;
+
+  @Column({ type: 'text' })
+  content: string;
```

Comment:
From what I understand from product, this is optional which makes `contentHtml` optional as well

Reply:
But that is the whole type right?
What else is there just a title alone?